src/platform/unittest/TlsTest.cpp

```diff
@@ -165,7 +205,19 @@ struct TlsTest : public ::testing::TestWithParam<bool>
                 &TlsContext::TlsClientCallbacks,
                 &ClientSecConfigClientCertNoCertValidation,
                 OnSecConfigCreateComplete));
-        ASSERT_NE(nullptr, ClientSecConfigClientCertNoCertValidation );
+        ASSERT_NE(nullptr, ClientSecConfigClientCertNoCertValidation);
+#endif
+
+#ifndef _WIN32
```

Comment:
Shouldn't this work with Windows OpenSSL too? But as a bigger issue, I would prefer to add Windows CAPI support in parallel. It shouldn't be too bad. I believe we already have some code to do it elsewhere.

Reply:
Yes, it should. I did not do any testing on Windows. I can take a look when adding CAPI calls. It would be nice to do it as separate PR.

Reply:
I'm fine with doing CAPI in another PR, but I'd like to get OpenSSL cross platform working in this one; so the `#ifndef _WIN32` won't work. You would have to add a new define (something line `QUIC_DISABLE_TICKET_KEY_TESTS`) to just disable the test code for this feature.

Reply:
I added QUIC_DISABLE_PFX_TESTS to guard the tests.

For CAPI we should be able to use. 
```
HCERTSTORE PFXImportCertStore(
  CRYPT_DATA_BLOB *pPFX,
  LPCWSTR         szPassword,
  DWORD           dwFlags
);
```
I'll need to do some more testing on life cycle for the HCERTSTORE.